src/accessibility/RovingTabIndex.tsx

```diff
@@ -43,6 +43,22 @@ import { FocusHandler, Ref } from "./roving/types";
  * https://developer.mozilla.org/en-US/docs/Web/Accessibility/Keyboard-navigable_JavaScript_widgets#Technique_1_Roving_tabindex
  */
 
+// <input> types where it's safe to use the arrow keys to navigate over to the next item.
+// i.e. it's ok to press the down arrow on a radio button to move to the next radio.
+// But it's not ok to press the down arrow on a <input type="text"> to move away because
+// the down arrow should move the cursor to the end of the input.
+const NAVIGABLE_INPUT_TYPES = [
+    'radio',
+    'checkbox',
+];
+
+// Check for form elements which utilize the arrow keys for native functions
+// like many of the text input varieties.
+export function checkInputtableElement(el: HTMLElement): boolean {
+    const inputUsesArrowkeys = (el.tagName === 'INPUT' && !NAVIGABLE_INPUT_TYPES.includes(el.getAttribute('type')));
+    return inputUsesArrowkeys || el.matches("textarea, select, [contenteditable=true]");
+}
```

Comment:
Calling out new stuff since last review cycle. We're now sharing `checkInputtableElement(el)` between `RovingTabIndex` and `ContextMenu` to use the same logic on whether to react on Arrow keys for certain form elements.

e.g. We want to be able to `Arrow` key between all of the radio and checkbox options. But we don't want `Arrow` key to navigate from an `<input type="text">` or `<textarea>`